mycroft/skills/main.py

```diff
@@ -49,23 +49,35 @@
 start_clock = time.time()
 
 DEBUG = Configuration.get().get("debug", False)
-skills_config = Configuration.get().get("skills")
-BLACKLISTED_SKILLS = skills_config.get("blacklisted_skills", [])
+skills_config = Configuration.get().get("skills", {})
 PRIORITY_SKILLS = skills_config.get("priority_skills", [])
-SKILLS_DIR = '/opt/mycroft/skills'
-
-installer_config = Configuration.get().get("SkillInstallerSkill")
+installer_config = Configuration.get().get("SkillInstallerSkill", {})
 MSM_BIN = installer_config.get("path", join(MYCROFT_ROOT_PATH, 'msm', 'msm'))
 
-MINUTES = 60  # number of seconds in a minute (syntatic sugar)
+MINUTES = 60  # number of seconds in a minute (syntactic sugar)
+
+
+def get_skills_dir():
+    skills_dir = Configuration.get().get("skills", {})\
+        .get("directory", '~/.mycroft/jarbas_skills')
```

Comment:
The default skills dir should probably be something neutral...or maybe not have a default at all letting it error if the directory isn't set in the config.

Reply:
the directory would probably be set on the enclosure config, perhaps the default could just be ~/skills ? not sure if erroring out if the best thing to do, it should work out of the box on first run before users even look at configs